core/src/components/badge/badge.ionic.scss

```diff
@@ -0,0 +1,29 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./badge";
+
+// Ionic Badge
+// --------------------------------------------------
+
+:host {
+  --padding-start: #{globals.$ionic-space-100};
+  --padding-end: #{globals.$ionic-space-100};
+  --padding-top: #{globals.$ionic-space-0};
+  --padding-bottom: #{globals.$ionic-space-0};
+
+  display: inline-flex;
+
+  align-items: center;
+  justify-content: center;
+
+  font-size: globals.$ionic-font-size-400;
+  font-weight: globals.$ionic-font-weight-medium;
```

Comment:
Should this be in `:host(.badge-small)` since the font-size increases/decreases based on the size?

Reply:
I didn't move font-weight because it doesn't change regardless of size.

Reply:
Sorry I meant to comment on the `font-size` line